File: Posts/Studying/Effective_Objective-C_2.0/Effective_Obj-C_Demo/Effective_Obj-C_Demo/Computer.m
//
//  Computer.m
//  Effective_Obj-C_Demo
//
//  Created by 买明 on 2019/3/19.
//  Copyright © 2019 kingcos. All rights reserved.
//

#import "Computer.h"

@implementation Computer
- (instancetype)init
{
    self = [super init];
    if (self) {
        self->_name = @"COMPUTER";
    }
    return self;
}

@end

File: Posts/Practice/Type_Introspection_and_Reflection/Type_Introspection_and_Reflection/Type_Introspection_and_Reflection/Swift/Computer.swift
//
//  Computer.swift
//  Type_Introspection_and_Reflection
//
//  Created by 买明 on 2019/4/11.
//  Copyright © 2019 kingcos. All rights reserved.
//

import Foundation

struct Computer {
    var system: String
    var memorySize: Int
    
    let run: () -> Void = {
        print(#function)
    }
    
    let runWithParam: (String) -> Void = { param in
        print("param: \(param)")
    }
}
